Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
 > Can be completed at the same time as [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] or [[XC.10 Lost Procedures and Diversion to Alternates]].
 
 ### Lesson Elements
+##### Handling Emergencies
```

Comment:
Reading through these, I wonder if it's better to have each debug flow end with "then run the <checklist_name> checklist". I worry there's too many details in here, while specific planes may have specific differences.

Exceptions to might be things like "emergency descent at speed which prevents combustion" -- basically things that explain the 'why' part of common checklist items